docs/setup-nginx-proxy-with-minio.md

```diff
@@ -62,20 +72,20 @@ If you want to serve web-application and Minio from the same nginx port then you
  }
 ```
 
-## 4. Recipe Steps
+## <a name="runnginx"></a>4. Run `NGINX`
 
-### Step 1: Start Minio server.
+### Start Minio Server
 
 ```sh
 minio server /mydatadir
```

Comment:
We should standardize on server syntax, I think most of the other docs use /data.

Reply:
@deekoder , @eco-minio : this is resolved in the latest commit. Please see https://github.com/jamesneimanconsulting/cookbook/blob/patch-6/docs/setup-nginx-proxy-with-minio.md.